cpp/arcticdb/storage/library.hpp

```diff
@@ -168,7 +168,8 @@ class Library {
     }
 
     std::string name() {
-        return library_path_.to_delim_path();
+        auto lib_name = storages_->name();
+        return lib_name;
```

Comment:
Why not just `return storages_->name();`?

Reply:
This way it is easier to debug (at least in VSCode) and shouldn't really make a difference in a release build